src/UID2.Client/Uid2ClientHelper.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace UID2.Client
+{
+    internal class Uid2ClientHelper
+    {
+        private readonly string _endpoint;
+        private readonly string _authKey;
+        private readonly byte[] _secretKey;
+
+        private readonly HttpClient _client = new HttpClient();
+
+        public Uid2ClientHelper(string endpoint, string authKey, string secretKey)
+        {
+            _endpoint = endpoint;
+            _authKey = authKey;
+            _secretKey = Convert.FromBase64String(secretKey);
+
+        }
+
+        internal static string GetAssemblyNameAndVersion()
+        {
+            var version = "5.5.0";  //ensure this is kept in sync with UID2.Client.nuspec
```

Comment:
maybe add a comment in UID2.Client.nuspec to remind people to update here as well?

Reply:
will do, although this is changing v soon due to Kat's changes